app/controllers/job_postings_controller.rb

```diff
@@ -1,2 +1,37 @@
 class JobPostingsController < ApplicationController
+  def index
+    @job_posting = JobPosting.new
```

Comment:
Are you sure you want to create a new JobPosting instance in the page that should be displaying all job postings?